src/components/item/Presenter.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './Presenter.scss';
+
+export const Item = ({ props }) => {
+  debugger;
+  return (
+    <div className="item" data-testid={`item-${props.id}`}>
+      <img data-testid={`item-image-${props.id}`} src={props.image} alt={props.title} />
+      <div className={`item-info`}>
+        <div className="item-title">
+          <span id={`item-title-${props.title}`}><strong>{props.title}</strong></span>
```

Comment:
Once there are multiple usages of a key (for example, `props.title`), it's probably a good time to start object restructuring to make it easier to change later

Reply:
To add to what Francesca says, you can add `const { title, price, endDate } = props` before your return, and then you won't have to reference `props.price`, `props.title`, etc in multiple places